shared/ui/Stream/icons8-data.json

```
@@ -98,8 +98,8 @@
 		"name": "gitlab",
 		"width": 16,
 		"height": 16,
-		"viewBox": "0 0 500 500",
-		"path": "<path id=\"path50\" class=\"st0\" d=\"M249.9,476.8L249.9,476.8l90.7-279.1H159.2L249.9,476.8L249.9,476.8z\"/><path id=\"path66\" class=\"st0\" d=\"M32.1,197.7L32.1,197.7L4.5,282.5c-2.5,7.7,0.2,16.2,6.8,21l238.5,173.3L32.1,197.7L32.1,197.7z\"/><path id=\"path74\" class=\"st0\" d=\"M32.1,197.7h127.1L104.6,29.6c-2.8-8.6-15-8.6-17.9,0L32.1,197.7L32.1,197.7z\"/><path id=\"path82\" class=\"st0\" d=\"M467.6,197.7L467.6,197.7l27.6,84.8c2.5,7.7-0.2,16.2-6.8,21L249.9,476.8L467.6,197.7L467.6,197.7z\"/><path id=\"path86\" class=\"st0\" d=\"M467.6,197.7H340.5l54.6-168.1c2.8-8.6,15-8.6,17.9,0L467.6,197.7L467.6,197.7z\"/>"
+		"viewBox": "93 93 194 194",
+		"path": "<defs><style>.cls-1{fill:#171321;}</style></defs><path class=\"cls-1\" d=\"M282.83,170.73l-.27-.69-26.14-68.22a6.81,6.81,0,0,0-2.69-3.24,7,7,0,0,0-8,.43,7,7,0,0,0-2.32,3.52l-17.65,54H154.29l-17.65-54A6.86,6.86,0,0,0,134.32,99a7,7,0,0,0-8-.43,6.87,6.87,0,0,0-2.69,3.24L97.44,170l-.26.69a48.54,48.54,0,0,0,16.1,56.1l.09.07.24.17,39.82,29.82,19.7,14.91,12,9.06a8.07,8.07,0,0,0,9.76,0l12-9.06,19.7-14.91,40.06-30,.1-.08A48.56,48.56,0,0,0,282.83,170.73Z\"/>"
```

This looks good, but we don't want to have hard-coded color styles and we don't need the inline class. Can you make this change? (see suggestion)

```suggestion
		"path": "<path d=\"M282.83,170.73l-.27-.69-26.14-68.22a6.81,6.81,0,0,0-2.69-3.24,7,7,0,0,0-8,.43,7,7,0,0,0-2.32,3.52l-17.65,54H154.29l-17.65-54A6.86,6.86,0,0,0,134.32,99a7,7,0,0,0-8-.43,6.87,6.87,0,0,0-2.69,3.24L97.44,170l-.26.69a48.54,48.54,0,0,0,16.1,56.1l.09.07.24.17,39.82,29.82,19.7,14.91,12,9.06a8.07,8.07,0,0,0,9.76,0l12-9.06,19.7-14.91,40.06-30,.1-.08A48.56,48.56,0,0,0,282.83,170.73Z\"/>"
```